yt_dlp/extractor/bannedvideo.py

```diff
@@ -0,0 +1,235 @@
+from __future__ import unicode_literals
+
+import json
+
+from .common import InfoExtractor
+
+from ..utils import (
+    try_get,
+    int_or_none,
+    url_or_none,
+    float_or_none,
+    unified_timestamp,
+    unified_strdate,
+)
+
+
+class BannedVideoIE(InfoExtractor):
+    _GRAPHQL_API = 'https://api.infowarsmedia.com/graphql'
+    _GRAPHQL_HEADERS = {
+        'Content-Type': 'application/json; charset=utf-8'
+    }
+    _GRAPHQL_GETVIDEO_QUERY = '''
+query GetVideo($id: String!) {
+    getVideo(id: $id) {
+        ...DisplayVideoFields
+        streamUrl
+        directUrl
+        unlisted
+        live
+        tags {
+            _id
+            name
+            __typename
+        }
+        sale {
+            _id
+            text
+            __typename
+        }
+        __typename
+    }
+}
+fragment DisplayVideoFields on Video {
```

Comment:
There is more metadata available such as `likes` etc. Why don't include them as well?